src/components/Wiki/WikiPage/WikiReferences.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react'
+import { CiteReference } from '@/types/Wiki'
+import {
+  Box,
+  Flex,
+  Heading,
+  SimpleGrid,
+  Tag,
+  Text,
+  Link,
+} from '@chakra-ui/react'
+
+interface WikiReferencesProps {
+  references: CiteReference[]
+}
+const WikiReferences = ({ references }: WikiReferencesProps) => {
+  const [currentLocationHash, setCurrentLocationHash] = useState('')
+  useEffect(() => {
+    const onHashChanged = () => {
+      setCurrentLocationHash(window.location.hash)
+    }
+    window.addEventListener('hashchange', onHashChanged)
+    return () => {
+      window.removeEventListener('hashchange', onHashChanged)
+    }
+  }, [])
+
+  if (references.length === 0) return null
+  return (
+    <Box borderTopWidth="1px" p={4}>
+      <Heading my={4} p={2} fontWeight="500">
+        REFERENCES
+      </Heading>
+      <SimpleGrid mb={8} columns={[1, 2, 3]} spacing={4}>
+        {references.map((ref, index) => (
+          <Flex
+            id={`cite-id-${ref.id}`}
+            bgColor={
```

Comment:
This is okay, but just thought you should know that `bg` is also okay here. doesn't have to be `bgColor`
```suggestion
            bg={